### PR TITLE
Mobile: Insert comment layout is too narrow and doesn't occupy full height

### PR DESCRIPTION
**Pull request by codewithvk**
Change-Id: Ibb6beff5659f5240ff2b49897e8a491f08933691

* Resolves:
* Target version: master 

Comment:

Screenshot from 2023-11-24 20-31-56

HyperLink: 
Screenshot from 2023-11-24 20-32-32

### CODE REVIEW

**Comment by codewithvk** [comment]
> This doesn't fix the bug. We want image to be image
> 
> The current PR affects negatively the whole UI, making multiple panels full height and sometimes even affecting bottom bar position:
> 
>  Peek.2023-12-14.16-16.mp4

Strange, Are you sure are you building the right branch? It's working fine for me, Here is the video.
[Screencast from 14-12-23 11:31:30 PM IST.webm](https://github.com/CollaboraOnline/online/assets/61119120/4d7ac6ff-5974-4d0e-a959-2e303b3a6fc0)

**Comment by codewithvk** [comment]
@pedropintosilva Can you please mention the height and width of the mobile screen that you tested in the above video?

File: browser/css/device-mobile.css [hunk 1]
```diff
@@ -681,12 +681,15 @@ td[id^='tb_spreadsheet-toolbar_item']:focus table.w2ui-button div.w2ui-icon, td[
...
+	position: fixed;
+	top: 0;
+	left: 0;
+	height: 100vh !important;
 }
 #mobile-wizard.popup #mobile-wizard-content,
 #mobile-wizard.popup .mobile-wizard-content {
 	display: flex;
-	align-items: center;
+	align-items: start;
```
Comment (pedropintosilva) [inline browser/css/device-mobile.css:692]: snackbar misalignment

**Review by pedropintosilva — CHANGES_REQUESTED** [review]
I have tested again, double checked on real phone and on emulators. 

This fixes the issue, making dialogs (popups) such as insert hyperlink and insert comment full height and fixing the layout. Thanks.

Just 2 last things I have noticed:

# Snackbar misaligned


https://github.com/CollaboraOnline/online/assets/65948705/692c74d3-2531-48f1-8b30-314a03ff23c5

How to test it: add a breakpoint in `showSnackbar` and then trigger the snackbar by stopping collabora online with ctrl + c  in the terminal. Or by directly calling somewhere `showSnackbar` like so:

```diff
diff --git a/browser/src/control/Control.UIManager.js b/browser/src/control/Control.UIManager.js
index 0448642bce..4ff02dbf98 100644
--- a/browser/src/control/Control.UIManager.js
+++ b/browser/src/control/Control.UIManager.js
@@ -971,20 +971,10 @@ L.Control.UIManager = L.Control.extend({
 		}
 	},
 
 	enterReadonlyOrClose: function() {
-		if (this.map.isEditMode()) {
-			// in edit mode, passing 'edit' actually enters readonly mode
-			// and bring the blue circle editmode button back
-			this.map.setPermission('edit');
-			var toolbar = w2ui['actionbar'];
-			if (toolbar) {
-				toolbar.uncheck('closemobile');
-				toolbar.uncheck('close');
-			}
-		} else {
-			window.onClose();
-		}
+		// show snackbar on mobile: tap pencil and then the checkmark to save and close
+		this.showSnackbar(_('Server is now reachable. We have to refresh the page now.'), _('RELOAD'), function() {console.debug('snackbar');});
 	},
 
 	onGoBack: function(popStateEvent) {
 		if (popStateEvent.state && popStateEvent.state.context) {

```


# Busypopup

Ideally we would do a follow up on another PR, where we make sure the parent of the busypopup gets another css class so we can then add it to the exclusion pattern on the css side

| Before this PR | After this PR |
| --- | --- |
| localhost_9980_browser_3c909e4300_debug html_file_path=_home_pedro_LibreOffice_online-23-05_test_data_hello-world odt(Pixel 5) | Screen Shot 2023-12-18 at 14 43 27 | 

# Snackbar

File: browser/css/device-mobile.css (hunk 1 above)
Comment (codewithvk) [reply browser/css/device-mobile.css:692]: Updated PR.

**Comment by codewithvk** [comment]
Created follow up PR to fix busy popup : ) PR: https://github.com/CollaboraOnline/online/pull/7891

**Review by pedropintosilva — APPROVED** [review]

**Pull request merged**